src/twisted/trial/reporter.py

```diff
@@ -104,6 +109,8 @@ def __init__(self):
         self.unexpectedSuccesses = []
         self.successes = 0
         self._timings = []
+        self._testStarted = 0
```

Comment:
the problem was that with 3.12.1 `startTest()` was no longer called for skipped tests... but `stopTest()` was called.

This is my take on the patch... it might not be the best option.

But I hope it's a start.